.github/workflows/upload-metadata-files.yml

```diff
@@ -0,0 +1,52 @@
+name: Upload any Changed Metadata Files
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - "airbyte-integrations/connectors/**/metadata.yaml"
+
+jobs:
+  upload-metadata-files-to-gcs:
+    name: "Upload changed Metadata Files to Prod Bucket"
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_HUB_PASSWORD: ${{ secrets.DOCKER_HUB_PASSWORD }}
+      DOCKER_HUB_USERNAME: ${{ secrets.DOCKER_HUB_USERNAME }}
```

Comment:
Can we move this to the step that requires it?

Reply:
Good call out!